AudioSystem/Editor/SequenceAudioConfigEditor.cs

```diff
@@ -0,0 +1,67 @@
+﻿using DUCK.Editor.Inspectors;
+using DUCK.Utils;
+using System;
+using System.Reflection;
+using UnityEditor;
+using UnityEngine;
+
+namespace DUCK.AudioSystem.Editor
+{
+	[CustomEditor(typeof(AudioConfigSequence))]
+	public class SequenceAudioConfigEditor : ArrayInspector
```

Comment:
This name needs to match the name change of `AudioConfigSequence` too.

Reply:
Done